docs/docs/20-usage/30-matrix-workflows.md

```diff
@@ -136,3 +137,7 @@ steps:
     when:
       platform: linux/arm*
 ```
+
+:::note
+If you want to control the architecture of a pipeline on a Kubernetes runner, see [the nodeSelector documentation of the Kubernetes backend](../30-administration/22-backends/40-kubernetes.md#nodeSelector).
```

Comment:
This is not related directly to the matrix builds.

Also you can control architecture of pipeline by labels, which is more canonical.

Reply:
I put it there as this is common use case if you're running on k8s and working with large matrices. And having a link to the respective docs explaining "how to" seems quite useful to me?